featuretools/tests/utils_tests/test_gen_utils.py

```diff
@@ -68,3 +69,16 @@ def test_list_semantic_tags():
     ft_semantic_tags = ft.list_semantic_tags()
     ww_semantic_tags = list_semantic_tags()
     assert ft_semantic_tags.equals(ww_semantic_tags)
+
+
+def test_camel_and_title_to_snake():
```

Comment:
can we include a snake case test case to confirm snake case passed in is unaltered?

Reply:
done